java/ql/lib/change-notes/2022-03-28-JumpStmt-as-superclass.md

```diff
@@ -0,0 +1,4 @@
+---
+category: fix
```

Comment:
Not really a "fix", but the other categories do not seem to fit much better either.